.github/workflows/build_dockers_workflow.yml

```
@@ -116,21 +184,128 @@ jobs:
+      packages: write
+
+    steps:
+      - name: Log in to Registries
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+          echo "${{ secrets.DOCKER_PASSWORD }}" | docker login ${{ secrets.DOCKER_PROVIDER }} -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+
+      - name: Pre for multi-arch image merge
+        env:
+          BMATRIX: "${{ needs.builds_envs_setup.outputs.matrix }}"
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2076:warning:22:47: Remove quotes from right-hand side of =~ to match as a regex rather than literally

(shellcheck)